sdk/src/main/java/io/dapr/actors/ActorProxyToAppAsyncClient.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * Interface to invoke actor methods.
+ */
+interface ActorProxyToAppAsyncClient {
```

Comment:
This can move to io.dapr.actors.client package. Fine for now, it can be moved when ActorProxy class is added.